deployment/ccip/changeset/testhelpers/messagingtest/helpers.go

```diff
@@ -116,10 +127,16 @@ func getLatestNonce(tc TestCase) uint64 {
 		}, tc.SourceChain, tc.Sender)
 		require.NoError(tc.T, err)
 	case chain_selectors.FamilySolana:
-		// var nonceCounterAccount ccip_router.Nonce
-		// err = common.GetAccountDataBorshInto(ctx, solanaGoClient, nonceEvmPDA, config.DefaultCommitment, &nonceCounterAccount)
-		// require.NoError(t, err, "failed to get account info")
-		// require.Equal(t, uint64(1), nonceCounterAccount.Counter)
+		ctx := context.Background()
+		client := tc.Env.SolChains[tc.DestChain].Client
+		// TODO: solcommon.FindNoncePDA expected the sender to be a solana pubkey
+		chainSelectorLE := solcommon.Uint64ToLE(tc.DestChain)
+		noncePDA, _, err := solana.FindProgramAddress([][]byte{[]byte("nonce"), chainSelectorLE, tc.Sender}, tc.OnchainState.SolChains[tc.DestChain].Router)
+		require.NoError(tc.T, err)
+		var nonceCounterAccount ccip_router.Nonce
+		// we ignore the error because the account might not exist yet
+		_ = solcommon.GetAccountDataBorshInto(ctx, client, noncePDA, solconfig.DefaultCommitment, &nonceCounterAccount)
+		latestNonce = nonceCounterAccount.Counter
```

Comment:
Wait are nonces a thing on Solana? I thought everything was out of order execution? Would this always be zero?

Reply:
Yeah I added support to this function but then realized it doesn't make sense. I forget how this fits together but there are still nonce accounts for the other direction (`svm->` rather than `->svm`)

Reply:
I'll probably just replace that section with a no-op/fail